nixos/modules/services/networking/wireguard.nix

```diff
@@ -419,6 +483,116 @@ let
         '';
       };
 
+  peerUpdateSecurity = {
```

Comment:
I always wonder for these large swathes of config what reasoning went into them. A comment would be good.

Reply:
I've added a comment mentioning that those settings come from `systemd-analyze security`.